test/tst_get_variables_by_attributes.py

```diff
@@ -0,0 +1,46 @@
+import unittest
+
+import netCDF4
+
+class VariablesByAttributesTests(unittest.TestCase):
+
+    def setUp(self):
+        netcdf_file = ('http://geoport.whoi.edu/thredds/dodsC/'
```

Comment:
I would strongly suggest creating a netCDF file to test instead. In my experience, remote DAP servers are not always reliable. This is especially annoying for packagers which run your unit tests to verify that they built your project successfully.